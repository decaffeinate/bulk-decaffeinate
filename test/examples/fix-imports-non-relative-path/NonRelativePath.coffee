# This actually references something in node_modules.
NonRelativePath = require 'NonRelativePath'

console.log NonRelativePath.foo

module.exports.foo = 3
