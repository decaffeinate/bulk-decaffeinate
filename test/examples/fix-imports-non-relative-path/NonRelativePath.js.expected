// TODO: This file was created by bulk-decaffeinate.
// Sanity-check the conversion and remove this comment.
// This actually references something in node_modules.
import NonRelativePath from 'NonRelativePath';

console.log(NonRelativePath.foo);

export let foo = 3;
